src/libraries/System.Security.Cryptography.Pkcs/src/System/Security/Cryptography/Pkcs/CmsSigner.cs

```diff
@@ -255,15 +255,15 @@ internal void CheckCertificateValue()
             {
                 if (IncludeOption == X509IncludeOption.EndCertOnly)
                 {
-                    certs.Add(Certificate);
+                    certs.Add(Certificate!);
```

Comment:
I see we use `Certificate!` in other places; we know it's not null because...?

Reply:
By this point, we've already used it... provided the signer type isn't NoSignature.